src/patternfly/components/TabContent/examples/tab-content-example.hbs

```diff
@@ -0,0 +1,12 @@
+{{#> tab-content tab-content--IsActive="true" tab-content--attribute='id="tab1-panel" aria-labelledby="tab1"'}}
```

Comment:
It might be worth updating the `aria-controls=""` in the tab component to match the id of each of these sections.

Reply:
Good call. I'll also remove the tab content sections currently being used in the tabs component examples.